### PR TITLE
Fix keypad and tab navigation issues in exercises

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Prevents the container divs around the keypad element from capturing click events, allowing the check button behind to still be clicked
* Ensures that when an answer is submitted - either through the 'check' button being pressed, or by 'enter' being pressed within the perseus question, that the 'next' button is then focused if the question is completed

## References
Bug reported on slack
Issue observed while manually testing fix

## Reviewer guidance
Ensure that the keypad is still completely usable.
Ensure that the 'Check' button can be clicked while the keypad is open.
Ensure that using keyboard navigation to press the check button keeps the next button focused if the answer was correct.
Ensure that pressing enter while the question is in focus and correct causes the next button to display focused.

----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue
Comment (marcellamaki) [inline kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue]: Is adding `onClick` specifically here allowing the click events to propagate properly?

Comment (rtibbles) [reply kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue]: Hrm, that might be unnecessary - it was definitely one of the things I tried. Will double check.

Comment (rtibbles) [reply kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue]: Yes, it is unnecessary, removing.

Comment (rtibbles) [reply kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue]: Updated.

**Review by marcellamaki — COMMENTED** [review]
Code changes seem reasonable to me -- will wait for Radina's QA approval

**Comment by radinamatic** [comment]
- Ensure that the keypad is still completely usable. ✔️ 
- Ensure that the 'Check' button can be clicked while the keypad is open. ✔️ 
- Ensure that using keyboard navigation to press the check button keeps the next button focused if the answer was correct. ✔️ 
- Ensure that pressing enter while the question is in focus and correct causes the next button to display focused. ✔️ 

Tested on a practice quiz too, both buttons are keyboard navigable, and mouse  navigable, even when _obscured_ by the keypad:


https://github.com/user-attachments/assets/82b5335f-b3e7-426d-846a-f8f937c42c0f

**Review by radinamatic — APPROVED** [review]
LGTM! 👍🏽 💯 :shipit:

**Review by marcellamaki — APPROVED** [review]

**Pull request merged**
